repos/fdbt-site/src/pages/ticketConfirmation.tsx

```diff
@@ -237,6 +238,69 @@ export const buildPointToPointPeriodConfirmationElements = (ctx: NextPageContext
     return confirmationElements;
 };
 
+export const buildFlatFarePriceByDistanceConfirmationElements = (
+    ctx: NextPageContextWithSession,
+): ConfirmationElement[] => {
+    const confirmationElements: ConfirmationElement[] = [];
+    const serviceInformation = getSessionAttribute(ctx.req, SERVICE_LIST_ATTRIBUTE) as ServiceListAttribute;
+    const services = serviceInformation ? serviceInformation.selectedServices : [];
+    const pricePerDistance = getSessionAttribute(ctx.req, PRICING_PER_DISTANCE_ATTRIBUTE);
+    const operatorAttribute = getSessionAttribute(ctx.req, OPERATOR_ATTRIBUTE);
+    const opName = operatorAttribute?.name ? `${operatorAttribute.name} ` : '';
```

Comment:
I dont think this can ever be undefined by this point, so we can remove the checks to see if its not there etc.

Reply:
was taken from another part of code - causing type script errors if removed